docs/copied-from-beats/command-reference.asciidoc

```diff
@@ -0,0 +1,581 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/command-reference.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
+
+// These attributes are used to resolve short descriptions
+
+:global-flags: Also see <<global-flags,Global flags>>.
+
+:export-command-short-desc: Exports the configuration or index template to stdout
+:help-command-short-desc: Shows help for any command
+:modules-command-short-desc: Manages configured modules
+:run-command-short-desc: Runs {beatname_uc}. This command is used by default if you start {beatname_uc} without specifying a command
+:setup-command-short-desc: Sets up the initial environment, including the index template, Kibana dashboards (when available), and machine learning jobs (when available)
+:test-command-short-desc: Tests the configuration
+:version-command-short-desc: Shows information about the current version
+
+
+[[command-line-options]]
+=== {beatname_uc} commands
+
+{beatname_uc} provides a command-line interface for running the Beat and
+performing common tasks, like testing configuration files and loading
+dashboards. The command-line also supports <<global-flags,global flags>>
+for controlling global behaviors.
+
+ifeval::["{beatname_lc}"!="winlogbeat"]
+
+[TIP]
+=========================
+Use `sudo` to run the following commands if:
+
+* the config file is owned by `root`, or
+* {beatname_uc} is configured to capture data that requires `root` access
```

Comment:
this is a bit weird for APM Server, but i guess this could be taken to mean if you set it up to listen on port < 1025, you'll need to run it as root.